mlir/include/mlir/IR/PatternMatchAction.h

```diff
@@ -0,0 +1,20 @@
+#ifndef MLIR_IR_PATTERNMATCHACTION_H
+#define MLIR_IR_PATTERNMATCHACTION_H
+
+#include "mlir/IR/Action.h"
+
+namespace mlir {
+struct ReplaceOpAction : public tracing::ActionImpl<ReplaceOpAction> {
+  using Base = tracing::ActionImpl<ReplaceOpAction>;
+  ReplaceOpAction(ArrayRef<IRUnit> irUnits, ValueRange replacement);
+  static constexpr StringLiteral tag = "op-replacement";
```

Comment:
```suggestion
  static constexpr StringLiteral tag = "ReplaceOpAction";
```

This suggestion applies only if this identifier is used for debugging purposes, so that filtering by the name of the action would work.

Reply:
I was following the naming from the [documentation](https://github.com/Xilinx/llvm-aie/blob/a1da90b27083ff18773b3ac293a8c665137efa80/mlir/docs/ActionTracing.md#L65)